main.tf

```diff
@@ -147,6 +149,11 @@ resource "github_branch_protection" "main" {
   }
 
   push_restrictions = var.push_restrictions
+  lifecycle {
+    ignore_changes = [
+      required_pull_request_reviews["pull_request_bypassers"]

```

Comment:
What happens if we change this after the first apply, will the changes still apply?

Reply:
Good catch, It won't apply the new changes. So ignore_changes option  doesn't resolve the issue